docs/installation/harvester-configuration.md

```diff
@@ -10,26 +20,26 @@ server_url: https://someserver:6443
 token: TOKEN_VALUE
 os:
   ssh_authorized_keys:
-  - ssh-rsa AAAAB3NzaC1yc2EAAAADAQAB...
-  - github:username
+    - ssh-rsa AAAAB3NzaC1yc2EAAAADAQAB...
+    - github:username
   hostname: myhost
   modules:
-  - kvm
-  - nvme
+    - kvm
+    - nvme
   sysctl:
     kernel.printk: "4 4 1 7"
     kernel.kptr_restrict: "1"
   dns_nameservers:
-  - 8.8.8.8
-  - 1.1.1.1
+    - 8.8.8.8
+    - 1.1.1.1
   ntp_servers:
-  - 0.us.pool.ntp.org
-  - 1.us.pool.ntp.org
+    - 0.us.pool.ntp.org
+    - 1.us.pool.ntp.org
   wifi:
-  - name: home
-    passphrase: mypassword
-  - name: nothome
-    passphrase: somethingelse
+    - name: home
+      passphrase: mypassword
+    - name: nothome
+      passphrase: somethingelse
```

Comment:
this would work without the prepose spaces.

Reply:
solved